packages/experiments/src/components/ExtendedPicker/BaseWellPicker.Props.ts

```diff
@@ -0,0 +1,90 @@
+import { IBaseFloatingPickerProps } from './BaseFloatingPicker.Props';
+import * as React from 'react';
+import { IPickerItemProps, BaseAutoFill, IBasePicker, IInputProps } from 'office-ui-fabric-react/lib/Pickers';
+import { BaseFloatingPicker } from './BaseFloatingPicker';
+
+export interface IBaseWellPicker<T> extends IBasePicker<T> {
+  /** Forces the picker to resolve */
+  forceResolve?: () => void;
+}
+
+// Type T is the type of the item that is displayed
+// and searched for by the people picker. For example, if the picker is
+// displaying persona's than type T could either be of Persona or Ipersona props
+export interface IBaseWellPickerProps<T> {
+  componentRef?: (component?: IBaseWellPicker<T>) => void;
+
+  /**
+   * Function that specifies how the selected item will appear.
+   */
+  onRenderItem?: (props: IPickerItemProps<T>) => JSX.Element;
+
+  /**
+   * Initial items that have already been selected and should appear in the people picker.
+   */
+  defaultSelectedItems?: T[];
+
+  /**
+   * A callback for when the selected list of items changes.
+   */
+  onChange?: (items?: T[]) => void;
+
+  /**
+   * A callback for when the user put focus on the picker
+   */
+  onFocus?: React.FocusEventHandler<HTMLInputElement | BaseAutoFill>;
+
+  /**
+   * A callback for when the user moves the focus away from the picker
+   */
+  onBlur?: React.FocusEventHandler<HTMLInputElement | BaseAutoFill>;
+
+  /**
+   * ClassName for the picker.
+   */
+  className?: string;
+
+  /**
+   * The floating picker type
+   */
+  floatingPickerType?: new (props: IBaseFloatingPickerProps<T>) => BaseFloatingPicker<T, IBaseFloatingPickerProps<T>>;
```

Comment:
This should be required, or there should be a floatingPicker

Reply:
Thanks Jon. Just to be clear, are you suggesting that I integrate the the FloatingPicker into the BasePicker or create a component that extends the BasePicker (that lives in experiments project) and use it there?